src/main/resources/config.yml

```diff
@@ -1,5 +1,5 @@
 # URL to request form to claim exclusive Hack Club stickers
-claim-stickers-url: "https://example.com/claim-stickers"
+claim-stickers-url: "http://hack.af/mc-stickers"
```

Comment:
i think this was set to an example, so the url wasnt public :see_no_evil: 
so if someone runs a copy on their own server (for whatever reason)...

Reply:
oops, i forgot about that